compiler/ccgexprs.nim

```diff
@@ -347,7 +347,8 @@ proc genAssignment(p: BProc, dest, src: TLoc, flags: TAssignmentFlags) =
     else:
       useStringh(p.module)
       linefmt(p, cpsStmts,
-           "memcpy((void*)$1, (NIM_CONST void*)$2, sizeof($1[0])*$1Len_0);$n",
+           #"memcpy((void*)$1, (NIM_CONST void*)$2, sizeof($1[0])*$1Len_0);$n",
+           "$1 = $2;$n",
```

Comment:
all other Nim types of varargs are not processes through this `else` branch. they are processed somewhere else.
because of bug #4799, this `else` branch never executed anymore.
the above code also not a C arrays assignment, but plain pointer assignment.
the transform part already transform the AST into single pointer assignment, not a sequence of pointers anymore.
because of this reasons, the memcpy no longer needed, at least we haven't encountered other use case of this `else` branch besides simple pointer assignment.

Reply:
perhaps a clear annotation/comment for this `else` branch sufficient, until another bug comes up?